test_db_model.py

```diff
@@ -189,7 +190,7 @@ def test_host_schema_timezone_enforced():
         "reporter": "test",
     }
     with pytest.raises(ValidationError) as excinfo:
-        _ = HostSchema(strict=True).load(host)
+        _ = HttpHostSchema(strict=True).load(host)
```

Comment:
Here, we can use _MqHostSchema_ (as the primary one) too.

```suggestion
        _ = MqHostSchema(strict=True).load(host)
```